delfin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -79,6 +117,14 @@ def do_call(self, url, data, method,
 
         try:
             res = func(url, **kwargs)
+        except requests.exceptions.SSLError:
```

Comment:
Can't we capture requests.exceptions.SSLError in driver.api.py for each driver interfaces and retry after relaod_certificate(). by this way we can make this opration to to the driver API layer instead if driver implemenation.